src/styles/buttons.scss

```diff
@@ -0,0 +1,46 @@
+@use './colors.scss';
+@use './fonts.scss';
+
+$btn-color-disabled: #69718c;
+$btn-bg-active: #ed6c39;
+
+button {
+  cursor: pointer;
+  padding: 5px 24px;
+  height: 40px;
+  border-radius: 12px;
```

Comment:
is there reason this is not in relative units?

Reply:
I usually keep the fonts in relative units and other things change only when we have a design for bigger screens as it not always looking good when you proportionally scale padding and border-radius. That's why I keep it static.